docs/mine-hnt/validators/mainnet/deployment-guide.mdx

```diff
@@ -0,0 +1,484 @@
+---
+id: validators-mainnet-deployment-guide
+title: Validator Deployment Guide - Mainnet
+sidebar_label: Deployment Guide - Mainnet
+slug: /mine-hnt/validators/mainnet/deployment-guide
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+This guide will cover the end-to-end process of deploying a Helium Validator on Mainnet. It assumes you're starting from scratch. At the end, you'll have a Helium Validator deployed on the Network.
+
+
+:::warning
+
+Although staking HNT has officially been enabled on Mainnet, Validators are not yet managing the Consensus Group - and therefore are not yet earning HNT. This will be enabled approximately 10-14 days after Mainnet reaches 100 staked Validators.
```

Comment:
```suggestion
Although initial staking has officially been enabled on Mainnet, Validators are not yet elected into the Consensus Group - and therefore are not yet earning HNT. This will be enabled at a later date, after the core developers believe it's safe to activate the chain variables. The current expectation is that this will be possible about 10-14 days after Mainnet reaches 100 staked Validators.
```